tox.ini

```diff
@@ -38,6 +38,8 @@ norecursedirs = .* docs requirements
 [testenv]
 deps = 
     django32: Django>=3.2,<4.0
+    django40: Django>=4.0,<4.1
```

Comment:
Why are we adding a tox environment for 4.0?  The upgrade page doesn't mention it: https://openedx.atlassian.net/wiki/spaces/AC/pages/3767926815/Django+4.2+Upgrade+Plan